app/org/sagebionetworks/bridge/stormpath/StormpathAccountDao.java

```diff
@@ -80,6 +81,13 @@ public void setStudyService(StudyService studyService) {
         this.studyService = studyService;
     }
     
+    public StormpathAccountDao() {
+        BridgeConfig config = BridgeConfigFactory.getConfig();
+        String url = config.getStormpathApplicationHref().trim();
+        String[] parts = url.split("/");
+        applicationId = parts[parts.length-1];
+    }
```

Comment:
This is a weird one because we're actually pulling the identifier out of the URL. Let me try and clean up.